_posts/2016-04-11-growing-developers-at-greenfield.md

```diff
@@ -0,0 +1,24 @@
+
+### Think Process, Not Product
+When you’re just starting out, it’s easy to look around at more experienced developers and feel overwhelmed by how much there is to learn. One of the best pieces of advice that I got early on was to not stress about how I stack up against everyone else and to just focus on my own learning one day at a time. To help myself stay focused on what’s important, I’ve kept a journal of what I’m learning each day, and recently, we’ve started documenting our TIL’s in a GitHub repo as a team. It’s helpful to have a record of what I’m learning not only as a reference for future me to look back on but also as a way to pause and recognize my successes and reflect on what I could do better tomorrow. If you spend all your energy focused on what you don’t know, it’s easy to [downplay the progress you're making](http://larahogan.me/donuts/). But If you concentrate on the *process* of learning something new each day, those small gains will add up to big wins over time. [All great developers were beginners](https://changelog.com/189/) at some point, and they achieved developer greatness one day a time, just like you.
+
+### Start Before You're Ready
+There’s a common misconception about learning, which assumes that learning and doing happen in two discrete steps, one after the other. Beginners often fall victim to this way of thinking because it can feel safer to focus on reading books, working through tutorials, and conceptual learning without getting one’s hands dirty. The truth, however, is that we learn by doing, and as software developers we learn to code by writing code. Becoming a better software developer is an iterative process, much like software development itself. When you execute your ideas, you discover what you really know and what you need to understand better. Each time you build something new, you learn something new. And there will always be more to learn, so the idea that you need to master a given language or technology before you begin is just not true. Instead, choose a side project to work on and troubleshoot as you go. Read the documentation. Dive into the source code. Ask for help on Slack or Stack Overflow. [Start before you're ready](http://sethgodin.typepad.com/seths_blog/2016/03/show-your-work.html?utm_source=feedburner&utm_medium=feed&utm_campaign=Feed%3A+typepad%2Fsethsmainblog+%28Seth%27s+Blog%29) because if you wait until you feel ready, you may be waiting a long time.
```

Comment:
was going to wait for the 80 char thing before commenting... but oh well. Can you remove the utm_source params and stuff from this URL?